### PR TITLE
Include SLE products into the CCE tooling for auto assignment

### PR DESCRIPTION
**Pull request by ggbecker**
#### Description:

- Include SLE products into the CCE tooling for auto assignment.
- CCE files were renamed to match the product name (`sles12` -> `sle12`; `sles15` -> `sle15`).
- Added new line to the CCE files which were missing.
- Added documentation for the `add-cce` subcommand of `fix_rules.py` utility. 
- Remove broken link about CCE syntax.

#### Rationale:

- CCE files from other distros are integrated into our automated tooling for CCE assignment.

@rumch-se FYI

### CODE REVIEW

File: docs/manual/developer/05_tools_and_utilities.md [hunk 1]
```diff
@@ -140,6 +141,30 @@ For example:
 Note that it is generally good practice to commit all changes prior to running
 one of these commands and then commit the results separately.
 
+#### How to automatically assign CCEs with the `add-cce` sub-command
+
+First you need to make sure that that the `rule_dirs.json` exists, run the following to create it:
```
Comment (jan-cerny) [inline docs/manual/developer/05_tools_and_utilities.md:146]:
```suggestion
First, you need to make sure that that the `rule_dirs.json` exists, run the following command to create it:
```

**Review by jan-cerny — APPROVED** [review]
looks amazing

**Comment by ggbecker** [comment]
@anivan-suse FYI

**Comment by anivan-suse** [comment]
> @anivan-suse FYI

Thanks for the update! @teacup-on-rockingchair FYI as well :)

**Pull request merged**